planning/behavior_path_avoidance_module/src/utils.cpp

```diff
@@ -756,11 +756,21 @@ size_t findPathIndexFromArclength(
 std::vector<UUID> concatParentIds(const std::vector<UUID> & ids1, const std::vector<UUID> & ids2)
 {
   std::vector<UUID> ret;
-  for (const auto id2 : ids2) {
```

Comment:
Previously, if the ids2 is empty, return ids also becomes empty even if the ids1 is **NOT** empty.